src/plugins/mpris/item.h

```diff
@@ -0,0 +1,48 @@
+// albert - a simple application launcher for linux
+// Copyright (C) 2014-2015 Manuel Schneider
```

Comment:
Thanks for giving the copyright :D Please make sure all files have the GPL licence header and keep the copyright.